lib/graphql/schema.ts

```diff
@@ -527,6 +527,12 @@ export type Mutation = {
   updateTransactionSplits: Transaction;
   /** Delete transaction splits */
   deleteTransactionSplits: Transaction;
+  /** Create transaction asset */
+  createTransactionAsset: CreateTransactionAssetMutationResult;
+  /** Create transaction asset */
+  finaliseTransactionAssetUpload: TransactionAsset;
+  /** Create transaction asset */
+  deleteTransactionAsset: MutationResult;
```

Comment:
These comments should probably not always be the same. Problem comes from backend.

Reply:
:+1: good catch, will update now.